src/index.html

```diff
@@ -17,6 +29,82 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
```

Comment:
The `alt` attribute of the image should describe the image content more specifically. For example, if it's the logo of Moyo, the alt attribute could be 'Moyo company logo'.